docs/reset-mfa.md

```diff
@@ -1,4 +1,23 @@
-# Reset MFA
+# Reset multifactor authentication (MFA)
+
+TechPass users need to reset MFA if they:
+
+- Change their mobile phone.
+- Delete Authenticator app from their mobile phone.
+
+Users whose TechPass domain is ```techpass.gov.sg```,  will only have TechPass profile(account) and must [Reset TechPass MFA](reset-techpass-mfa-for-new-device) when needed.
+
+Users whose TechPass domain is not ```techpass.gov.sg``` will have two profiles(accounts) - SGGovt M365(WOG) and TechPass. Such users need do the following:
+
+- [Reset WOG(SGGovt M365) MFA](reset-security-verification-for-wog-account).
+
+- [Create service request](https://go.gov.sg/techpass-sr) to remove the MFA of their TechPass profile.
+
+- When the request is processed successfully, the support team sends an email with instructions on how to reset MFA for the TechPass account.
```

Comment:
```suggestion
- When the request is processed successfully, you will be able to register a new TechPass MFA profile when you try to sign in again.
```

Reply:
hmm, the reason i have phrased it in such a manner because 
1. This is what is happening
2. The email tells what to do next .

If I says you will be able to register a new TechPass MFA profile, user will ask then how to do it? For that, if I link user to https://docs.developer.tech.gov.sg/docs/techpass-user-guide/onboard-public-officers-using-non-se-machines?id=step-4-onboard-to-techpass

it does not have the correct starting point. This was written from the onboarding part.  (we need to revisit if step 4 in PO onbaording has to match with the steps in the email)

 I felt the one on the email has it clearly.